ui/packages/catalog-ui-search/src/main/webapp/extension-points/table-export/table-export.tsx

```diff
@@ -56,16 +56,21 @@ export function buildCqlQueryFromMetacards(metacards: any, count: any) {
   }
   return `(${queryParts.join(' OR ')})`
 }
-const visibleData = (selectionInterface: any) =>
+const visibleData = (size: number, selectionInterface: any) =>
   buildCqlQueryFromMetacards(
     selectionInterface.getActiveSearchResults().toJSON(),
-    null
+    size
   )
+const limit = 10
 const allData = (selectionInterface: any) =>
   selectionInterface.getCurrentQuery().get('cql')
-function getCqlForSize(exportSize: string, selectionInterface: any) {
-  return exportSize === 'visible'
-    ? visibleData(selectionInterface)
+function getCqlForSize(
+  exportType: string,
+  size: number,
+  selectionInterface: any
+) {
+  return exportType == 'custom' && size <= limit
```

Comment:
I think this should be `exportType !== 'all'`, otherwise visible will always go to the `allData` method

Reply:
good catch 👍

Reply:
good catch